### PR TITLE
Updated the upgrade-ops-manager pipeline to allow customers to specif…

### PR DESCRIPTION
**Pull request by rahulkj**
…y the disk type while deploying Ops Manager

### CODE REVIEW

File: upgrade-ops-manager/vsphere/params.yml [hunk 1]
```diff
@@ -36,3 +36,4 @@ ntp: CHANGEME
 opsman_network: CHANGEME
 iaas_type: vsphere
 git_private_key:
+om_disk_type: thick # use thick or thin disk provisioning
```
Comment (krishicks) [inline upgrade-ops-manager/vsphere/params.yml]: This should be changed to thin given the old default was thin.

The param should also be `opsman_disk_type` given the other params format.

File: tasks/deploy-opsman-vm/task.yml [hunk 2]
```diff
@@ -44,3 +44,4 @@ params:
   OPSMAN_SSH_PASSWORD:
   OPSMAN_NETWORK:
   OPSMAN_VM_FOLDER:
+  OM_DISK_TYPE:
```
Comment (krishicks) [inline tasks/deploy-opsman-vm/task.yml]: This should be `OPSMAN_DISK_TYPE` to keep in line with the other params.

**Comment by rahulkj** [comment]
updated

**Comment by abbyachau** [comment]
Thanks @rahulkj. We've prioritised this pr.

**Pull request merged**